providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/job.py

```diff
@@ -95,7 +95,9 @@ class KubernetesJobOperator(KubernetesPodOperator):
         compatibility.
     """
 
-    template_fields: Sequence[str] = tuple({"job_template_file"} | set(KubernetesPodOperator.template_fields))
+    template_fields: Sequence[str] = tuple(
+        sorted({"job_template_file"} | set(KubernetesPodOperator.template_fields))
+    )
```

Comment:
Seems easier to do this:
```suggestion
    template_fields: Sequence[str] = tuple("job_template_file", *KubernetesPodOperator.template_fields)
```